website/docs/cloud-docs/users-teams-organizations/permissions.mdx

```diff
@@ -16,7 +16,9 @@ description: >-
 
 HCP Terraform's access model is team-based. In order to perform an action within an HCP Terraform organization, users must belong to a team that has been granted the appropriate permissions.
 
-The permissions model is split into organization-level, project-level, and workspace-level permissions. Additionally, every organization has a special team named "owners", whose members have maximal permissions within the organization.
+The permissions model is split into organization-level, project-level, and workspace-level permissions. These permissions are additive, meaning that a user on a team is allowed to perform the most permissive action that has been granted on a resource, regardless of the level at which it was granted. For example, if a team has a workspace "read" permission at the organization-level, and they have admin access on a single workspace, users on that team can still apply runs -- the organization-level permission doesn't take precedence over the workspace permissions. When configuring permissions at a particular level, it's important to remember that that team may have permissions above or below the hierarchy that grant a higher level of permission than what's currently being configured. A team's _effective permissions_ are the sum of the permissions that have been granted to them across the entire organization hierarchy.
+
+Additionally, every organization has a special team named "owners", whose members have maximal permissions within the organization.
```

Comment:
Can we remove this sentence because 
```suggestion
Every organization also has a special team called the **Owners** team, whose members have the maximum available permissions within the organization.
```

Reply:
Oh yeah, I didn't even see that just below! Thanks for the callout.